### ISSUE TITLE
Dask Dataframe - groupby using a list of column names

### ISSUE THREAD

**Issue by thequackdaddy** [dask/dask#1169]
Per the [documentation](http://dask.pydata.org/en/latest/dataframe-api.html#dask.dataframe.DataFrame.groupby) you should be able to `groupby` using just a "tuple/list of column names".

But it doesn't seem to work. 

```python
In [4]: dask.__version__
Out[4]: '0.9.0'

In [5]: import dask.dataframe as ddf
   ...: import pandas as pd
   ...: x = pd.DataFrame({'a': [1, 1, 2, 2, 3], 'b': [1, 2, 3, 4, 5]})
   ...: x.groupby(['a']).sum()
   ...: 
   ...: x = ddf.from_pandas(x, chunksize=2)
   ...: x.groupby(['a']).sum().compute()
Traceback (most recent call last):

  File "<ipython-input-5-42f283b0c734>", line 7, in <module>
    x.groupby(['a']).sum().compute()

  File "C:\Anaconda3\lib\site-packages\dask-0.9.0-py3.5.egg\dask\base.py", line 37, in compute
    return compute(self, **kwargs)[0]

  File "C:\Anaconda3\lib\site-packages\dask-0.9.0-py3.5.egg\dask\base.py", line 110, in compute
    results = get(dsk, keys, **kwargs)

  File "C:\Anaconda3\lib\site-packages\dask-0.9.0-py3.5.egg\dask\threaded.py", line 57, in get
    **kwargs)

  File "C:\Anaconda3\lib\site-packages\dask-0.9.0-py3.5.egg\dask\async.py", line 488, in get_async
    raise(remote_exception(res, tb))

TypeError: unorderable types: list() > int()

Traceback
---------
  File "C:\Anaconda3\lib\site-packages\dask-0.9.0-py3.5.egg\dask\async.py", line 267, in execute_task
    result = _execute_task(task, data)
  File "C:\Anaconda3\lib\site-packages\dask-0.9.0-py3.5.egg\dask\async.py", line 249, in _execute_task
    return func(*args2)
  File "C:\Anaconda3\lib\site-packages\dask-0.9.0-py3.5.egg\dask\dataframe\groupby.py", line 211, in <lambda>
    agg = lambda df: aggfunc(df.groupby(level=levels))
  File "C:\Anaconda3\lib\site-packages\pandas\core\generic.py", line 3748, in groupby
    sort=sort, group_keys=group_keys, squeeze=squeeze)
  File "C:\Anaconda3\lib\site-packages\pandas\core\groupby.py", line 1428, in groupby
    return klass(obj, by, **kwds)
  File "C:\Anaconda3\lib\site-packages\pandas\core\groupby.py", line 345, in __init__
    level=level, sort=sort)
  File "C:\Anaconda3\lib\site-packages\pandas\core\groupby.py", line 2320, in _get_grouper
    elif level > 0:
```

**Comment by jcrist** [comment]
Thanks for the bug report - this should be fixed in #1171.

**Comment by thequackdaddy** [comment]
Wow you are fast. I'll try it out!
